.pre-commit-config.yaml

```diff
@@ -22,3 +22,9 @@ repos:
     hooks:
       - id: prettier
         types_or: [yaml]
+  - repo: https://github.com/citation-file-format/cffconvert
+    rev: "054bda51dbe278b3e86f27c890e3f3ac877d616c"
```

Comment:
I don't like this either, but it's what they documented. Opened issue: https://github.com/citation-file-format/cffconvert/issues/375